tgui/packages/tgui/interfaces/ExodroneConsole.tsx

```diff
@@ -0,0 +1,793 @@
+            )}
+            <Stack.Item grow />
+          </Stack>
+        </Stack.Item>
+      </Stack>
+    </Section>);
+};
+
+const AdventureScreen = (props, context) => {
+  const { act, data } = useBackend<ExodroneConsoleData>(context);
+  const {
+    adventure_data,
+  } = data;
+  return (
+    <Section
+      fill
+      title="Exploration"
+      buttons={<DroneStatus />}>
+      <Stack>
+        <Stack.Item>
+          <BlockQuote style={{ "white-space": "pre-wrap" }}>{adventure_data.description}</BlockQuote>
+        </Stack.Item>
+        <Stack.Divider />
+        <Stack.Item>
+          <img
+            src={adventure_data.raw_image ? adventure_data.raw_image : resolveAsset(adventure_data.image)}
+            height="100px"
+            width="200px"
+            style={{
+              '-ms-interpolation-mode': 'nearest-neighbor',
+            }} />
+          <Stack vertical>
+            <Stack.Divider />
+            <Stack.Item grow />
+            {!!adventure_data.choices && adventure_data.choices.map(choice => (
+              <Stack.Item key={choice.key}>
+                <Button
+                  fluid
+                  content={choice.text}
+                  textAlign="center"
+                  onClick={() => act('adventure_choice', { choice: choice.key })} />
+              </Stack.Item>
+            ))}
+            <Stack.Item grow />
+          </Stack>
+        </Stack.Item>
+      </Stack>
+    </Section>);
+};
+
+const DroneScreen = (props, context) => {
+  const { act, data } = useBackend<ExodroneConsoleData>(context);
+  const {
+    drone_status,
+    event,
+  } = data;
+  switch (drone_status) {
+    case "busy":
+      return (<TimeoutScreen />);
+    case "idle":
+    case "travel":
+      return (<TravelTargetSelectionScreen />);
+    case "adventure":
+      return (<AdventureScreen />);
+    case "exploration":
+      if (event) { return (<EventScreen />); }
+      else { return (<ExplorationScreen />); }
+  }
+};
+
+const ExodroneConsoleContent = (props, context) => {
+  const { act, data } = useBackend<ExodroneConsoleData>(context);
+  const {
+    drone,
+    drone_name,
+    drone_log,
+  } = data;
+
+  if (!drone) { return (<DroneSelectionSection />); }
+
+  return (
+    <Stack fill vertical>
+      <Stack.Item grow>
+        <Stack vertical fill grow={2}>
+          <Stack.Item grow>
+            <Stack fill>
+              <Stack.Item>
+                <EquipmentGrid />
+              </Stack.Item>
+              <Stack.Item grow basis={0}>
+                <DroneScreen />
+              </Stack.Item>
+            </Stack>
+          </Stack.Item>
+        </Stack>
+      </Stack.Item>
+      <Stack.Item height={10}>
+        <Section title="Drone Log" fill scrollable>
+          <LabeledList>
+            {drone_log.map((log_line, ix) => (
+              <LabeledList.Item key={log_line} label={`Entry ${ix + 1}`}>
+                {log_line}
+              </LabeledList.Item>))}
```

Comment:
i think this is a mistake i did, but:
![image](https://user-images.githubusercontent.com/40974010/111882956-a6f22c80-8975-11eb-9a0f-329921492bc7.png)